### PR TITLE
Feature/header - v2

### PR DESCRIPTION
**Pull request by sirius651**
**Pull Request Summary**

> Describe what changes this pull request makes to the repository

**Check list**
- [ ] contains breaking changes
- [x] adds new feature
- [ ] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes

**This pull request makes the following changes:**

**Adds**
스크린샷 2022-03-24 오후 4 19 43

- only available on dark theme. 
- regarding icon-base with tailwindcss, will refactor as next

- [ ] (ex: add user list)

### CODE REVIEW

File: src/components/header/AccountButton.vue [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+});
+</script>
+
+<style scoped>
+.btn--account {
+  display: flex;
+  height: 32px;
+  flex-direction: row;
+  align-items: center;
+  background: #2c3335;
```
Comment (impelcrypto) [inline src/components/header/AccountButton.vue:81]: Let's import the color variable from [quasar.variables.scss](https://github.com/AstarNetwork/astar-apps/blob/feature/header/src/css/quasar.variables.scss#L3)

File: src/components/header/MetaUpdateButton.vue [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+}
+.btn--update:disabled {
+  background: #d3d6dc;
+}
+.m-btn--update {
+  padding-left: 10px;
+  width: 32px;
+  height: 32px;
+  background: #2c3335;
+  color: #ff5621;
```
Comment (impelcrypto) [inline src/components/header/MetaUpdateButton.vue:93]: This one too 
https://github.com/AstarNetwork/astar-apps/blob/feature/header/src/css/quasar.variables.scss#L10

**Review by impelcrypto — CHANGES_REQUESTED** [review]
Let's import the color variable from `quasar.variables.scss`
https://github.com/AstarNetwork/astar-apps/blob/feature/header/src/css/quasar.variables.scss

File: src/components/header/MetaUpdateButton.vue (hunk 2 above)
Comment (sirius651) [reply src/components/header/MetaUpdateButton.vue]: Fixed

**Comment by Kahonnohak** [comment]
@sirius651 
It looks lovely thanks! May I ask 2 adjustments please?

1. Do we want to show the wallet(polkadot) next to the network? It is clear which wallet you are connected as wallet button is next to it?
2. the dividing line has 2px, could you make it 1px and the colour is #191D1F.

**Comment by sirius651** [comment]
> 2\. #191D1F.

Fixed above. could you check again? @Kahonnohak

**Review by hoonsubin — APPROVED** [review]
In terms of features, I think it is working well.
LGTM

**Review by impelcrypto — APPROVED** [review]

**Comment by Kahonnohak** [comment]
Sorry I forgot to check the mobile view, there are things that I would like to reproduce the design but I know we don't have time so we will skip but we need the Astar logo here.  However I'd like to  confirm when the sidebar is implemented by Roy on Monday. In April (after portal release) I would like to come back to this header structure as well as background blur.
スクリーンショット 2022-03-27 1 20 33

**Comment by Kahonnohak** [comment]
As I said above, no need changes on the header apart from the logo on mobile view and could you also check the text(Assets) that is really #F7F7F8? I can see that but it looks darker. And I believe I'm waiting for the navi for mobile too🙏

**Pull request merged**